stock_picking_accounting_date/views/stock_picking_views.xml

```diff
@@ -9,7 +9,9 @@
                 <field name="show_accounting_date" invisible="1" />
                 <field
                     name="accounting_date"
-                    attrs="{'invisible': [('show_accounting_date', '=', False)]}"
+                    attrs="{'invisible': [('show_accounting_date', '=', False)],
+                    'readonly': [('show_accounting_date', '=', True),
```

Comment:
@rinaldifirdaus Any reason to have this `('show_accounting_date', '=', True)`?

Reply:
If we don't define ('show_accounting_date', '=', True), the field with string consumable accounting date will also become readonly as far as i test.

Reply:
so the purpose is to make it readonly only when storable product on the transfers. the one with string "Accounting Date".